#!/usr/bin/python3

def raise_exception():
    c = 3 / "boy"
    return c
